docs/guide/setup/creating_ligand_networks.rst

```diff
@@ -2,51 +2,48 @@
 
 Defining the Ligand Network
 ===========================
-A :class:`.LigandNetwork` is a set of small molecules connected by mappings of two small molecules.
-One example for such a network could be a set of drug candidates, that should be ranked using alchemical transformations.
-The :class:`.LigandNetwork` is a tool that is used to orchestrate the free energy calculations to efficiently
-compute a ligand ranking.
-It is of course possible to calculate all possible :class:`.Transformation` defined by all possible :class:`.AtomMappings`
-connecting all :class:`.SmallMoleculeComponent` with a ''maximal network'' (using :func:`.generate_maximal_network`),
-but it is much more efficient to use a network with less transformations like a ''radial network'' (also known as a star map, using :func:`.generate_radial_network`)
-or a ''minimimal spanning network'' (using :func:`.generate_minimal_spanning_network`).
+A :class:`.LigandNetwork` is a network where nodes are :class:`.SmallMoleculeComponent`\ s and edges are :class:`.LigandAtomMapping`\ s.
```

Comment:
What's possibly missing here is some kind of exposition that tells users that edges are not always expected. I.e. you can have a ligand networks with disconnected sections.